lib/mini-lightbox.js

```diff
@@ -1,6 +1,45 @@
 // MiniLightbox
 // </> with <3 by Ionică Bizău
-(function (root) {
+(function () {
```

Comment:
> I wouldn't advise for that to be honest, since that would break the usage for everyone that installed the module to date.

That's what I'm saying to bump it to `2.0.0`, because it's breaking change.

Browserify will take care to wrap it nicely, anyways.